open_rarity/resolver/rarity_providers/rarity_sniffer.py

```diff
@@ -52,10 +53,14 @@ def get_all_ranks(contract_address: str) -> dict[str, int]:
         )
 
         if response.status_code != 200:
+            try:
+                response_json = response.json()
+            except JSONDecodeError:
+                response_json = ""
```

Comment:
```suggestion
                response_json = "Failed to decode JSON Response"
```
Or it only fails when it's "" ?